### DISCUSSION TITLE
Modelleerprincipes - U4. Informatiemodel is voldoende flexibel en uitbreidbaar

### DISCUSSION THREAD

**Discussion by pmaria** [Geonovum/disgeo-imsor#56] [General] [answered]
Modelleerprincipe [[U4]](https://geonovum.github.io/disgeo-imsor/modelleerprincipes/#u4-informatiemodel-is-voldoende-flexibel-en-uitbreidbaar) - Het informatiemodel (IMSOR) moet voldoende flexibel, uitbreidbaar en inkrimpbaar zijn wat betreft objecttypen, eigenschappen en het al dan niet verplicht zijn hiervan.

Discussie uit expertgroepsessie:

> - Michel: Wanneer je data hebt wordt dit lastiger, want dan moet het bv backwards-compatible zijn, etc.
> - Het is ook handig om iets te zeggen over versie-management, werken met vastgestelde versies.
> - permanent BETA is niet een handige term => levende standaard?

**Comment by lvdbrink** [comment, marked as answer]
Over versiemanagement kunnen we zeggen dat we werken met 
- werkversies
- consultatieversies
- versie ter vaststelling
- vastgestelde versie

En we kunnen verwijzen naar het [Generiek beheerplan standaarden comform BOMOS - Geonovum](https://www.geonovum.nl/documents/geonovum-generiek-beheerplan-standaarden-v11pdf) 

"Levende standaard" vind ik een goede term. Dit is wel iets nieuws voor Geonovum. W3C kent tegenwoordig een procedure voor levende standaarden

Wel is het natuurlijk zo dat een informatiemodel wel flexibel te maken is, terwijl deze flexibiliteit op inwin-, databeheer- en softwareniveau vaak lastig te hanteren is. Daar kunnen wij vanuit de informatiemodellering niet alles aan doen. Het zou goed zijn dit punt (wat betekent een flexibel en uitbreidbaar informatiemodel in de praktijk?) eens concreter uit te diepen met bv de architecten van de SOR en/of in een praktijkproef.

**Comment by lennartvanbergen** [comment]
Spelen nog 2 dingen mee: 

- Versies van informatiemodellen van afzonderlijke registraties di de SOR in samenhang brengt. De SOR beweegt mee. Mag geen bottlekneck zijn hierin. 
- Vertalingen van registraties naar SOR moeten meegenomen worden in het versiebeheer 
-

**Comment by lvdbrink** [comment]
U4 is vastgesteld in [expertgroep 8 juli](https://github.com/Geonovum/disgeo-imsor/blob/master/overleg/08-07-2021-aantekeningen.md).
